src/modules/sih/sih.hpp

```diff
@@ -274,7 +293,7 @@ class Sih : public ModuleBase<Sih>, public ModuleParams, public px4::ScheduledWo
 	// parameters defined in sih_params.c
 	DEFINE_PARAMETERS(
 		(ParamInt<px4::params::IMU_GYRO_RATEMAX>) _imu_gyro_ratemax,
```

Comment:
So the sih rate on realtime operation is set by IMU_GYRO_RATEMAX, right?

Reply:
Yes, that's right. In lockstep, it is the minimum of `IMU_GYRO_RATEMAX` `IMU_INTEG_RATE`. This is to ensure that there is an output after each lockstep